sbin/link-balancer.in

```diff
@@ -25,35 +25,31 @@
 #       See the file COPYING for details.
 #
 
-VERSION='$Id$'
-PROGRAM_FILE="${0}"
-PROGRAM_DIR="${0%/*}"
-if [ "$PROGRAM_DIR" = "$0" ]; then PROGRAM_DIR="."; fi
+VERSION='@PACKAGE_VERSION@'
+PROGRAM_FILE="$(@READLINK@ $0)"
+PROGRAM_FILE="${PROGRAM_FILE:-$0}"
+PROGRAM_DIR="$(@DIRNAME@ "$PROGRAM_FILE")"
```

Comment:
same, this can be acquired form single source file.
there is no much sense in @READLINK@ and @DIRNAME@ as these are part of coreutils, so it should be save to use them while looking for the template config file.